test_app/settings.py

```diff
@@ -45,6 +45,7 @@
     'ansible_base.jwt_consumer',
     'ansible_base.resource_registry',
     'test_app',
+    'django_extensions',
```

Comment:
I still can't use shell_plus after adding this. What else is needed?

Reply:
hmm works fine for me, do you have ipython installed? (should be in requirements_dev.txt)

Reply:
It worked for me only after adding `"django_extensions"` to test_app INSTALLED_APPS like AWX has

Reply:
wait, this has that, I guess I was on another branch, this should be good :+1: